gpflow/models/gpmc.py

```diff
@@ -62,7 +63,13 @@ def __init__(
             loc=to_default_float(0.0), scale=to_default_float(1.0)
         )
 
-    def log_likelihood(self, *args, **kwargs) -> tf.Tensor:
+    def log_posterior_density(self) -> tf.Tensor:
+        return self.log_likelihood() + self.log_prior_density()
```

Comment:
this appears to be redundant

Reply:
it's intentional! to be explicit for MCMC models